software/lamp-os/src/lamps/standard_lamp.cpp

```diff
@@ -73,18 +77,41 @@ void initBehaviors() {
   baseKnockoutBehavior = lamp::KnockoutBehavior(&base, 0, true);
   baseKnockoutBehavior.knockoutPixels = config.base.knockoutPixels;
 
+  // Initialize expressions
+  expressionManager.begin(&shade, &base);
+  expressionManager.loadFromConfig(config.expressions);
+
+  // Build behavior vector in priority order (lowest to highest)
+  // Behaviors run in sequence, so later ones override earlier ones
+  std::vector<lamp::AnimatedBehavior*> allBehaviors = {};
+
+  // Add expression behaviors (lowest priority - automated effects)
+  auto exprBehaviors = expressionManager.getBehaviors();
+  allBehaviors.insert(allBehaviors.end(), exprBehaviors.begin(), exprBehaviors.end());
+
+  // Add configurator behaviors (middle priority - UI preview, overrides expressions)
+  allBehaviors.push_back(&shadeConfiguratorBehavior);
+  allBehaviors.push_back(&baseConfiguratorBehavior);
```

Comment:
configurator should override dmx and social. Users should see what they're working on while they're in the web ui